src/Controller/Api/DatasetSubmissionController.php

```diff
@@ -406,4 +409,26 @@ public function validateUrlAction(int $id, Request $request, UrlValidation $urlV
 
         return $urlValidation->validateUrl($erddapUrl);
     }
+
+    /**
+     * Returns a list of global ingest folders.
+     *
+     * @Route(
+     *     "/api/dataset_submission_folder_list",
+     *     name="pelagos_api_get_folder_list_dataset_submission",
+     *     methods={"GET"},
+     *     defaults={"_format"="json"},
+     *     )
+     *
+     * @View()
+     *
+     * @return Response
+     */
+    public function getGlobalIngestFolders(IngestUtil $ingestUtil): Response
+    {
+        if (!($this->getUser() instanceof Account)) {
+            throw new AccessDeniedException('User is either not logged in or does not have an account');
```

Comment:
You can't tell from this this if the user has an account or not. So here it just means that the user is not logged in. 
The text could also say something like "Must be logged in".
```suggestion
            throw new AccessDeniedException('User is either not logged in');
```